client.go

```diff
@@ -496,6 +496,98 @@ func (client *gocloak) GetClient(token string, realm string, clientID string) (*
 	return &result, nil
 }
 
+// GetClientsDefaultScopes returns a list of the client's default scopes
+func (client *gocloak) GetClientsDefaultScopes(token string, realm string, clientID string) ([]ClientScope, error) {
```

Comment:
Please rewrite function to return pointers, not objects. Take a look at other functions, all functions return pointers now. For example: https://github.com/Nerzal/gocloak/blob/master/client.go#L555

```go
func (client *gocloak) GetClientsDefaultScopes(token string, realm string, clientID string) ([]*ClientScope, error) {
     var result []*ClientScope
```